cards/multilang-product-prominentvideo/component.js

```diff
@@ -54,6 +54,35 @@ class multilang_product_prominentvideoCardComponent extends BaseCard['multilang-
     };
   }
 
+  onMount() {
+    const videoSelector = '.js-HitchhikerProductProminentVideo-video';
+    const videoEl = this._container.querySelector(videoSelector);
+    if (!videoEl) {
+      return;
+    }
+    const addPlayer = api => {
+      api.addPlayer(videoEl, {
+        onPlay: () => this.onPlay()
+      });
+    };
+    if (this.youtubeUrl) {
+      HitchhikerJS.requireYoutubeAPI().then(addPlayer);
+    } else if (this.vimeoUrl) {
+      HitchhikerJS.requireVimeoAPI().then(addPlayer);
+    }
+  }
+
+  onPlay() {
+    const event = new ANSWERS.AnalyticsEvent('CTA_CLICK')
+      .addOptions({
+        verticalKey: this.verticalKey,
+        entityId: this.result._raw.id,
```

Comment:
nit: I'm wondering if we want to do some null checking here just to be extra careful. Maybe something like `this.result?._raw?.id`

Reply:
shouldn't hurt!